docs/concepts/managed-ui.mdx

```diff
@@ -1,71 +0,0 @@
----
-id: managed-ui
-title: Managed User Interface
----
-
-Ory Cloud provides a managed user interface (UI) by default per Ory Project. The Managed UI gives you the ability to integrate
```

Comment:
Should we park this content in a potential "user guide" section?

Reply:
This document is a collection of screenshots that bring no real value. Additionally, they'll soon be outdated with the managed UI update in the works. I think it's okay to just scrap it.